### PR TITLE
Create endpoints for getting candidates and creating a candidate

### PR DESCRIPTION
**Pull request by olafleur**

### CODE REVIEW

File: src/main/java/com/arcbees/bourseje/server/api/AdminResource.java [hunk 1]
```diff
@@ -62,4 +64,12 @@ public Response setVoteState(VoteState voteState) {
 
         return Response.ok().build();
     }
+
+    @POST
+    @Path(ResourcesPath.CANDIDATE)
+    public Response addCandidate(Candidate candidate) {
```
Comment (meriouma) [inline src/main/java/com/arcbees/bourseje/server/api/AdminResource.java]: I think the REST resources could be rewritten a bit to return objects instead of Response. You could also use the ResourceDelegate. Not sure if it's worth it for this project at this point though

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/bourseje/server/api/AdminResource.java (hunk 1 above)
Comment (Chris-V) [inline src/main/java/com/arcbees/bourseje/server/api/AdminResource.java]: That would do another showcase for sure :)

Comment (olafleur) [inline src/main/java/com/arcbees/bourseje/server/api/AdminResource.java]: Will do in another PR
